Tests/LibWeb/Text/input/canvas/global-composite-operation.html

```diff
@@ -0,0 +1,54 @@
+<script src="../include.js"></script>
+<script>
+    function colorStr(d) {
+        // Give some margin of error
+        const r = d[0] - d[0] % 5;
+        const g = d[1] - d[1] % 5;
+        const b = d[2] - d[2] % 5;
```

Comment:
This test fails on MacOs, the blending operations can have slightly different results on different systems.
The way I'm accounting for it here is problematic close values can still give different results (for example 170 -> 170 and 169 -> 165).

One way to fix this could be to store the expected results in the test and check that the difference is smaller than a threshold (similar to what WPT tests do), but this is not the best because it would make the test impossible to `--rebaseline` automatically.

I'm happy to receive a suggestion, if you can think of a better way